### DISCUSSION TITLE
I found new fun development board (sipeed m0s dock tinyml risc-v bl616)

### DISCUSSION THREAD

**Discussion by np1139059565** [micropython/micropython#12007] [Hardware & Peripherals] [answered]
Guys, I found a very small development board (sipeed m0s dock tinyml risc-v bl616), less than 10mm, and the performance is great, the main frequency reaches 320MHz, the main controller chip uses the RISC-V architecture, and the most important thing is that it supports wifi6 (the official website says so). But unfortunately I see that micropython doesn't have firmware for this board, I'm looking forward to experiencing the speed of wifi6!
Here's their wiki address: https://wiki.sipeed.com/hardware/zh/maixzero/m0s/m0s.html

**Comment by robert-hh** [comment, marked as answer]
Thanks Jennifer. As far as I recall, Sipeed provides it's own flavour of MicroPython, called MaixPy. I used it with a K210 board. And there is a discussion thread about Buffalo labs devices: https://github.com/orgs/micropython/discussions/10667

**Reply by np1139059565** [reply]
Thank you very much for the information, I will go and verify if it works, wait for my results~

**Comment by bidrohini** [comment]
This is the first time I am hearing about this board. Is there any tutorial available?
